server/routes/topics.js

```diff
@@ -37,6 +38,7 @@ const createTopicsRouter = ({ hubMenuService }) => {
         config,
       });
     } catch (e) {
+      logger.error(`Error loading topics: ${e.message}`);
```

Comment:
errors propagate up to here so doing the logging in one place... hopefully we could push this into a global error handler